<?xml version="1.0"?>
<launch>

  <!-- <include file="$(find nav_pumas)/config_files/launch/navigation_real.launch"/> -->
  <node pkg="human_detector"  name="human_detector" type="pointing_detector_server.py"/>  
  <node pkg="face_recog"  name="face_recog" type="face_recog_server.py"/>
  <node pkg="face_recog"  name="face_recog_deep" type="face_recog_server_deep.py"/>

  <node pkg="face_recog"  name="analyze_face_backgnd" type="analyze_face_backgnd_node"/>
  <node pkg="head"  name="mic_node" type="mic_node"/>
  <!-- <node pkg="task"        name="write_analyze_result" type="write_analyze_result.py"/> -->

  <!-- <node pkg="smach_viewer"  name="smach_viewer" type="smach_viewer.py"/>   -->
  <!-- <node pkg="segmentation"  name="segmentator_node" type="segment_floor.py"/>   -->

<include file="$(find speech_recog)/launch/ros_vosk.launch"/>

  
<!-- </include> -->
<!-- <include file="$(find pocketsphinx)/launch/pocketsphinx.launch"> -->
  <!-- </include> -->
<!--node pkg="task" name="receptionist" type="Receptionist_gr.py"/-->
</launch>

 
